### ISSUE TITLE
Clarification For Each Individual Element #1

### ISSUE THREAD

**Issue by Adel-Akbi** [med-material/TunnelGoalFittsRShiny#1]
Hello ✌️ @bastianilso,  I'v made the structure of the app and the main design of it, so I need more clarification for each element to be able to understand what they really do : 

1. About the first drop down which contains the mail address of each person, is it the address mail we enter on the first scene on Unity  (ConnectToTheArduino)? which is also stored within the DB 

   ![1](https://user-images.githubusercontent.com/45661981/71803858-255cfa00-3062-11ea-98ca-291800735c24.PNG)

2. In the TestDetails page, exactly in the drop down "Choose Test" there is a date/time with a specific time in "min" is that means the time of execution ? 

    ![2](https://user-images.githubusercontent.com/45661981/71804336-6e617e00-3063-11ea-8e83-0b5aa054ada2.PNG)

 
3. What the "Index of Difficulty" means ? 

    ![3](https://user-images.githubusercontent.com/45661981/71804489-e0d25e00-3063-11ea-8f61-83c5851e6445.PNG)

4. In the Compare Responder + Type page, the Input to compare are all the Input Responder/Input Type available within Unity or only these three ? 

   ![4](https://user-images.githubusercontent.com/45661981/71804895-0dd34080-3065-11ea-8eef-2a750cfe488a.PNG)
 
    Ps : There no Nose in the Input responder 🗡  ❌ 

5. What is the main purpose of these two Coefficients (a,b) ?

   ![5](https://user-images.githubusercontent.com/45661981/71805537-c9e13b00-3066-11ea-8c39-ca8685b10b20.PNG)

Thanks 🍰

**Comment by bastianilso** [comment]
>     1. About the first drop down which contains the mail address of each person, is it the address mail we enter on the first scene on Unity  (ConnectToTheArduino)? which is also stored within the DB
>        ![1](https://user-images.githubusercontent.com/45661981/71803858-255cfa00-3062-11ea-98ca-291800735c24.PNG)

Yes 👍  - that would be the E-mail column in the DB. If "Everyone" is selected in the dropdown, then we show data for everybody. If a specific E-mail is selected, you filter the data based on the chosen e-mail.
![image](https://user-images.githubusercontent.com/3967945/71805943-da45e580-3067-11ea-8807-d9fcb16de243.png)

>     2. In the TestDetails page, exactly in the drop down "Choose Test" there is a date/time with a specific time in "min" is that means the time of execution ?

Yes, that is the time of execution (SessionTime).


>     3. What the "Index of Difficulty" means ?
>        ![3](https://user-images.githubusercontent.com/45661981/71804489-e0d25e00-3063-11ea-8f61-83c5851e6445.PNG)
Index of Difficulty (ID) is a number you will need to calculate in your R code, based on the Distance (D) between targets and Width (W) between targets, with the formula below from [Wikipedia](https://en.wikipedia.org/wiki/Fitts%27s_law#Bits_per_second:_model_innovations_driven_by_information_theory):
![image](https://user-images.githubusercontent.com/3967945/71806301-bdf67880-3068-11ea-9170-651976f536f3.png)


>     4. In the Compare Responder + Type page, the Input to compare are all the Input Responder/Input Type available within Unity or only these three ?

Yes. First people choose Test type. Based on the  chosen Test type (Fx Fitts test), the program filters the database and finds what Input Type and what Input Responders have been used in this test. Each unique combination of Input Type and Input Responder you find, should be then be listed `$InputType and $InputResponder`.


>        Ps : There no Nose on the Input responder 🗡  ❌

You're right, just ignore that 😄 

>     5. What is the main purpose of these two Coefficients (a,b) ?
>        ![5](https://user-images.githubusercontent.com/45661981/71805537-c9e13b00-3066-11ea-8c39-ca8685b10b20.PNG)

You get the coefficients `a` and `b` from calculating in R the linear regression, see this on [wikipedia](https://en.wikipedia.org/wiki/Simple_linear_regression).
See the `lm()` function.

**Comment by Adel-Akbi** [comment]
@bastianilso, some additional questions : 

-Does the movement time (s) mean the 'DeltaTime' within the DB ?

-About the second point, does the time of execution is calculated in "s" or "min" ? If it's in (s) how can I get the SessionTime in 'min' ? 🌖  

-Do both diagrams are same ? Or each page (Test Details - Compare Test) has its own diagram to generate data? 

-So, there are both coefficients (a & b) for each 'Input to Compare' ? right? 🌝

**Comment by bastianilso** [comment]
> -Does the movement time (s) mean the 'DeltaTime' within the DB ?

Yes.

> -About the second point, does the time of execution is calculated in "s" or "min" ? If it's in (s) how can I get the SessionTime in 'min' ? waning_gibbous_moon

You're right, SessionTime is in seconds, lets keep time of execution also in seconds, because these tests seem usually pretty short.

> -Do both diagrams are same ? Or each page (Test Details - Compare Test) has its own diagram to generate data?

In Test Details diagram, you only show data from the chosen session (data points, maybe with one which is averaged across all rows in the session). In the background you can make a filled grey shape, indicating where data from others are (who used same input responder and input modality).

> -So, there are both coefficients (a & b) for each 'Input to Compare' ? right? full_moon_with_face

Yes. Coefficients are shown for each input which is checked off :ballot_box_with_check:

**Comment by Adel-Akbi** [comment]
@bastianilso 
I'm just wondering if that green point represents the 'Invalid Hit' ? or something else? 🍔 
![Captuxre](https://user-images.githubusercontent.com/45661981/71912542-fcc62480-3175-11ea-9fbd-92296b08908b.PNG)

**Comment by bastianilso** [comment]
@Adel-Akbi the green point is the point calculated by averaging across movement times, for the chosen test (index of difficulty stays the same, since the width/distance does not change within the test).

If youre interested, we could also show each individual movement time:
![image](https://user-images.githubusercontent.com/3967945/71963506-0db97900-31fc-11ea-8ca6-c1b05ab5a79c.png)

**Comment by hendrikknoche** [comment]
showing each movement time would be good. And maybe showing the regression line based on those dots. See an example her
![image](https://user-images.githubusercontent.com/1982879/71971914-ef5b7980-320b-11ea-83d6-4daa8d862270.png)
